src/components/Garden/ProposalDetail/ProposalDetail.js

```diff
@@ -171,143 +177,159 @@ function ProposalDetail({
                   `}
                 >
                   <div>
-                    <ProposalHeader proposal={proposal} />
-                    <h1
-                      css={`
-                        ${textStyle('title2')};
-                      `}
-                    >
-                      {name}
-                    </h1>
+                    {isAbstainProposal ? (
+                      <AbstainCardHeader proposal={proposal} />
+                    ) : (
+                      <ProposalHeader proposal={proposal} />
+                    )}
+
+                    {!isAbstainProposal ? (
+                      <>
+                        <h1
+                          css={`
+                            ${textStyle('title2')};
+                          `}
+                        >
+                          {name}
+                        </h1>
+
+                        <div
+                          css={`
+                            margin-top: ${1 * GU}px;
+                          `}
+                        >
+                          <TransactionBadge
+                            transaction={txHash}
+                            networkType={network.type}
+                            explorerProvider={network.explorer}
+                          />
+                        </div>
+
+                        <div
+                          css={`
+                            margin-top: ${2 * GU}px;
+                            grid-column: span 2;
+                            min-width: ${40 * GU}px;
+                            color: ${theme.contentSecondary};
+                          `}
+                        >
+                          {fundingProposal ? (
+                            <span>
+                              This proposal is requesting{' '}
+                              <strong>
+                                {formatTokenAmount(
+                                  requestedAmountConverted,
+                                  requestToken.decimals
+                                )}
+                              </strong>{' '}
+                              {requestToken.symbol} out of{' '}
+                              <strong>
+                                {formatTokenAmount(
+                                  commonPool,
+                                  requestToken.decimals
+                                )}
+                              </strong>{' '}
+                              {requestToken.symbol} currently in the common
+                              pool.
+                            </span>
+                          ) : (
+                            <span>
+                              This suggestion is for signaling purposes and is
+                              not requesting any {requestToken.symbol}
+                            </span>
+                          )}
+                        </div>
+                      </>
+                    ) : null}
```

Comment:
What you do think to get that block and put inside the else of first if? After Line 187? So you dont need repeat of IF and dont need else with null

Reply:
The comment as bellow. Prefer adding condition to sections.